### PR TITLE
change to whispering-docs-theme

### PR DESCRIPTION
**Pull request by sallyom**
Signed-off-by: Sally O'Malley <somalley@redhat.com>

@cooktheryan @iranzo 
Check this out, please! I prefer this theme. 

Note: The video on the main page we will replace with a microshift screen recording
Note: There are plenty of bugs/typos still

Install latest `hugo`, check this branch out, then 
`hugo serve` 

Is it possible to install hugo-extended for the CI? Hope so, if we like this theme, or similar, extended is required.

### CODE REVIEW

File: content/docs/building-microshift/index.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---
+title: 'Local Development'
```
Comment (sallyom) [inline content/docs/building-microshift/index.md:2]: Need to move this file properly, so the git authorship is maintained.

Comment (sallyom) [reply content/docs/building-microshift/index.md:2]: (this is copied from CONTRIBUTING.md)

**Comment by iranzo** [comment]
+1 on this Theme

Some things I found: it doesn't put data for OGP (twitter, keywords, etc, we should check if the theme has them or if not, add that part).

**Comment by iranzo** [comment]
Hugo extended via #8

File: content/docs/building-microshift/index.md (hunk 1 above)
Comment (iranzo) [reply content/docs/building-microshift/index.md:2]: we can git checkout the previous commits to get contributing and then move it to the folder

**Pull request merged**